src/main/java/com/graphhopper/converter/resources/ConverterResourceNominatim.java

```diff
@@ -0,0 +1,79 @@
+package com.graphhopper.converter.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import com.graphhopper.converter.api.GHResponse;
+import com.graphhopper.converter.api.NominatimEntry;
+import com.graphhopper.converter.api.OpenCageDataResponse;
+import com.graphhopper.converter.api.Status;
+import com.graphhopper.converter.core.Converter;
+import java.util.HashMap;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.*;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ * @author Robin Boldt
+ */
+@Path("/nominatim")
+@Produces(MediaType.APPLICATION_JSON)
+public class ConverterResourceNominatim {
```

Comment:
Actually I am not sure. The benefit of the current approach is, that different providers might use different url patterns and we can essentially clone their url pattern. I think we can use it as is for now and decide later when we have more experience with it.